plugin/src/main/java/at/helpch/chatchat/config/mapper/SoundMapper.java

```diff
@@ -0,0 +1,44 @@
+package at.helpch.chatchat.config.mapper;
+
+import net.kyori.adventure.key.Key;
+import net.kyori.adventure.sound.Sound;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.intellij.lang.annotations.Subst;
+import org.jetbrains.annotations.NotNull;
+import org.spongepowered.configurate.ConfigurationNode;
+import org.spongepowered.configurate.serialize.SerializationException;
+import org.spongepowered.configurate.serialize.TypeSerializer;
+import java.lang.reflect.Type;
+
+
+public class SoundMapper implements TypeSerializer<Sound> {
```

Comment:
Pretty sure Adventure already has Configurate mappers for Sound and other stuff. We can use those instead. I will be replying with the link when I Find it.

Reply:
Found it. <https://jd.adventure.kyori.net/serializer-configurate3/4.10.1/net/kyori/adventure/serializer/configurate3/package-summary.html> I believe it comes with the main adventure package. all you have to do is add it as a serializer here <https://github.com/HelpChat/ChatChat/blob/8ddaadb238f792be7fa2f6eba0b4ee0fa60b28df/plugin/src/main/java/at/helpch/chatchat/config/ConfigFactory.java#L86>

Reply:
That mapper is for components, not sounds